### DISCUSSION TITLE
Adding header and body script from the control panel?

### DISCUSSION THREAD

**Discussion by freshface** [studio1902/statamic-peak#256] [Ideas]
We frequently need to implement external javscript code for clients.
It would be nice if you can do is straight from the cp.

See example:

<img width="931" alt="Screenshot 2022-09-09 at 14 34 17" src="https://user-images.githubusercontent.com/103913/189351075-4a0b47a2-15cb-4db9-a9b1-320dd06e9ebf.png">

**Comment by robdekort** [comment]
Shouldn't these scripts 9/10 times be behind a cookie banner? In that case you have two options:

1. Load it via GTM (yikes)
2. Load them in via script fragments: https://github.com/studio1902/statamic-peak/pull/252

We _could_ think of something like this if you often have to embed non-gdpr-violating scripts.

**Reply by robdekort** [reply]
Was this any help to you @freshface or did you have other ideas?
